packages/grid_client/src/modules/tft.ts

```diff
@@ -5,10 +5,10 @@ import { CurrencyModel } from "./models";
 
 class TFTUSDConversionService {
   // TFT rate: 1 TFT = x USD
-  constructor(protected rate: number, private decimals = 2) {}
+  constructor(protected _rate: number, private decimals = 2) {}
```

Comment:
any special need for protected modifier? are you planning to extend that class and children to have access to it?